app-services/secrets/res/configuration.toml

```diff
@@ -52,17 +51,35 @@ Host = "localhost"
 Port = 27017
 Timeout = "30s"
 
+# Shared Secret Store
 [SecretStore]
   Host = 'localhost'
   Port = 8200
-  Path = '/v1/secret/edgex/appservice'
+  Path = '/v1/secret/edgex/edgex-appservice/'
   Protocol = 'http'
```

Comment:
Change to `https`

Reply:
im running Vault in dev mode, which runs the server in HTTP